src/docs/labels/dao/index.md

```diff
@@ -1,19 +1,18 @@
 ---
-title: DAO Label
+title: 
```

Comment:
missing title?

Reply:
Just in general decided to remove current info about this label (and exclude from labels index)